src/FavoritesContext.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useContext, useEffect} from 'react'
```

Comment:
```js
import React, { useState, useContext, useEffect **space**} from 'react'
```
I suggest using **https://prettier.io/** to avoid having to deal with these styling issues.